src/core/src/pso.rs

```diff
@@ -208,7 +208,7 @@ impl Descriptor {
 }
 
 /// A complete set of vertex buffers to be used for vertex import in PSO.
-#[derive(Copy, Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
Ok, I see. Please bump both version numbers then and make sure the new core is requested